Src/AutoFixture.NUnit3/InlineAutoDataAttribute.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using NUnit.Framework.Interfaces;
+
+namespace Ploeh.AutoFixture.NUnit3
+{
+    /// <summary>
+    /// This attribute acts as a TestCaseAttribute but allow incomplete parameter values, 
+    /// which will be provided by AutoFixture. 
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Method, AllowMultiple = true)]
+    [CLSCompliant(false)]
+    public class InlineAutoDataAttribute : AutoDataAttribute
```

Comment:
Oh I was suggesting AutoDataAttribute:InlineAutoDataAttribute. But that is  not consistent with the current implementation.

I figure it WilL be possible to compose testcases through reflection? Or better that AutoData returns a derived type of TestCase that somehow exposes certain values for composition. However I need to look harder and experiment.